iina/PlayerCore.swift

```diff
@@ -202,11 +202,10 @@ class PlayerCore: NSObject {
     getSelectedTracks()
   }
 
-  /** Set speed. A negative speed -x means slow by x times */
+  /** Set speed. */
   func setSpeed(_ speed: Double) {
-    let realSpeed = Utility.toRealSpeed(fromDisplaySpeed: speed)
-    mpvController.setDouble(MPVOption.PlaybackControl.speed, realSpeed)
-    info.playSpeed = realSpeed
+    mpvController.setDouble(MPVOption.PlaybackControl.speed, speed)
```

Comment:
Oh I mean, currently, pausing the video will reset the speed to normal (1x). So `setSpeed(0)` in `togglePause()` should be `setSpeed(1)` now.